public/static/locales/en/manageProjects.json

```diff
@@ -40,8 +37,9 @@
   "addCaption": "Add Caption",
   "yearOfAbandonment": "Approx. year of Abandonment",
   "firstTreePlanted": "First tree planted",
+  "labelRestorationStarted": "Restoration Started",
   "plantingDensity": "Planting Density",
-  "maxPlantingDensity": "Max Planting Density (optional)",
+  "maxPlantingDensity": "Max Planting Density",
```

Comment:
Mandatory fields are indicated with * in the label. All other fields are optional. 

Have changed the placeholder text here because it was a little too long and looked odd.